StRoot/StFttClusterMaker/StFttClusterMaker.h

```diff
@@ -24,11 +24,11 @@ class StFttClusterMaker: public StMaker {
     ~StFttClusterMaker();
 
 
-    int  Init();
-    int  InitRun( int );
-    int  FinishRun( int );
-    int  Finish();
-    int  Make();
+    Int_t  Init();
```

Comment:
Basic c-types (int, float, double, etc...) are preferred in makers.  Int_t, Float_t, etc... should be used in classes which are made persistent.

Reply:
See comment below, all changes related to FTT reverted since they should be handled in PR #569 .
Done with https://github.com/star-bnl/star-sw/pull/571/commits/6088f974a30af0141f460b3fdd3183ae4f7c66ec